shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -230,14 +230,14 @@ public boolean getIsSoftwareRenderingEnabled() {
   }
 
   /**
-   * Observatory URI for the VM instance.
+   * VM Service URI for the VM instance.
    *
    * <p>Its value is set by the native engine once {@link #init(Context, String[], String, String,
    * String, long)} is run.
    */
   @Nullable
-  public static String getObservatoryUri() {
-    return observatoryUri;
+  public static String getVMServiceUri() {
```

Comment:
@bkonyi The removal of this is breaking the `espresso` plugin: https://github.com/flutter/plugins/blob/main/packages/espresso/android/src/main/java/androidx/test/espresso/flutter/action/FlutterViewAction.java#L108

And is breaking the Flutter to plugins roller. 

Could we add the `getObservatoryUri` back and mark it as deprecated until `getVMServiceUri` makes it to stable?

Reply:
@bparrishMines if this issue is blocking rolls, please file a P0 on GitHub, and assign @bkonyi.

Reply:
Created https://github.com/flutter/flutter/issues/118896